cfgov/scripts/unpublish_closed_jobs.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import urllib2
+import requests
+import json
+import os
+
+from bs4 import BeautifulSoup
+
+from jobmanager.models.pages import JobListingPage
+
+logger = logging.getLogger(__name__)
+
+URL = os.environ.get('USAJOBS_ARCHIVE_URL')
+HEADERS = {
+    'Host': os.environ.get('USAJOBS_HOST'),
+    'User-Agent': os.environ.get('USAJOBS_USER'),
+    'Authorization-Key': os.environ.get('USAJOBS_API_KEY'),
+}
+
+
+def job_page_closed(link):
+    webpage = urllib2.urlopen(link)
```

Comment:
Minor thing, but if you `from urllib2 import urlopen` above and then just use `urlopen` here, it'll make porting to Python 3 a little bit easier when we do (`urlopen` is in `urllib.requests` in Python 3, so we can make `urlopen`'s import conditional).

Reply:
This should be updated in https://github.com/cfpb/cfgov-refresh/pull/3787/commits/2b8db04cfab056d2b00745a3d9ed954b3f4ff240